sdv/constraints/utils.py

```diff
@@ -104,3 +104,59 @@ def sigmoid(data, low, high):
     data = data * (high - low) + low
 
     return data
+
+
+def check_nans_row(row):
+    """Check for NaNs in a pandas row.
+
+    Outputs a concatenated string of the column names with NaNs.
+
+    Args:
+        row (pandas.Series):
+            A pandas row.
+    """
+    columns_with_nans = []
+    for column, value in row.items():
+        if pd.isna(value):
+            columns_with_nans.append(column)
+
+    if columns_with_nans:
+        return ', '.join(columns_with_nans)
+    else:
+        return 'None'
```

Comment:
Can we add a test case for this? As in add a unit test that has no nans and make sure everything behaves correctly there. We should also test that `transform` does not add the `.nan_component` column in this case